Documentation/spelling_wordlist.txt

```diff
@@ -128,6 +128,7 @@ allocateLoadBalancerNodePorts
 allocator
 allocators
 allowedConfigOverrides
+allowlist
```

Comment:
I'm surprised that `allowlist` and `denylist` weren't already in the word list. 😲